common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -4195,17 +4192,21 @@ static jboolean NativeCrypto_X509_check_issued(JNIEnv* env, jclass, jlong x509Re
     return static_cast<jboolean>(ret);
 }
 
-static void get_X509_signature(X509* x509, ASN1_BIT_STRING** signature) {
-    *signature = x509->signature;
+static const ASN1_BIT_STRING* get_X509_signature(X509* x509) {
+    const ASN1_BIT_STRING* signature;
+    X509_get0_signature(&signature, nullptr, x509);
```

Comment:
(I don't know why `X509_get0_signature` and `X509_CRL_get0_signature` have different argument orders. OpenSSL...)